twitchobserver/twitchobserver.py

```diff
@@ -304,14 +345,19 @@ def _process_server_messages(self, response):
                 event.nickname = nick
                 event._command = cmd
 
-                if cmd in ('JOIN', 'PART'):
+                if cmd in ('JOIN', 'PART', 'USERSTATE', 'ROOMSTATE'):
                     event.channel = args[1:]
 
-                elif cmd == 'PRIVMSG':
+                elif cmd in ('PRIVMSG', 'HOSTTARGET', 'NOTICE', 'USERNOTICE'):
                     channel, message = _privmsg_params_re.match(args).groups()
                     event.channel = channel
                     event.message = message
 
+                elif cmd == "WHISPER":
+                    recipient, message = _whisper_params_re.match(args).groups()
+                    event.recipient = recipient
```

Comment:
I don't think we need this attribute. It appears to always be the name of the observer. If this isn't the case let me know.